DANS-KNAW/easy-solr4files-index
# EASY-1320 Fixed solr fileitems core configuration

Fixes EASY-1320

#### When applied it will
* no longer produce `attr_` prefixed field names in the search results
* 
* 

#### Where should the reviewer @DANS-KNAW/easy start?

This is a manual cherry pick of https://github.com/DANS-KNAW/easy-dtap/pull/156
But combined the tasks that copy schema.xml and solrconfig.xml in vagrant.yml

#### How should this be manually tested?

* `mvn clean install && vagrant destroy -f && vagrant up`
* Check the result in your browser with `http://test.dans.knaw.nl:8983/solr/#/fileitems/query`, you should find nothing.
* `curl -X POST 'http://test.dans.knaw.nl/fileindex/init'`
  it should report one bag for the pdbs store.
* Repeat the check with the browser. Now you should find 9 file items.
  The field names don't have a `attr_` prefix but start with `easy_file_` or `easy_dataset_`

#### Related pull requests on github
repo                       | PR                | note
-------------------------- | ----------------- | ----
easy-dtap                      | TODO    | should have a new role for this new module and reuse `src/main/assembly/dist/install/fileitems/*`
